ui/component/page/view.jsx

```diff
@@ -63,6 +65,7 @@ function Page(props: Props) {
     location: { pathname },
   } = useHistory();
 
+  const theaterMode = renderMode === 'video' ? videoTheaterMode : false;
```

Comment:
This breaks theater mode for "audio" -- the Recommended section gets overlapped instead of re-flowed.